util/cdn.py

```diff
@@ -1,25 +1,42 @@
 """Turn local URLs into CDN URLs."""
-
 from nose.tools import set_trace
+import os, sys
+util_dir = os.path.split(__file__)[0]
+core_dir = os.path.split(util_dir)[0]
+
 import urlparse
-from s3 import S3Uploader
 
-def cdnify(url, cdns):
-    if not cdns:
-        # No CDNs configured
-        return url
+def cdnify(_db, url):
     scheme, netloc, path, query, fragment = urlparse.urlsplit(url)
 
+    # TODO: Find a better way to import from model in core.util when
+    # using the parent directory (e.g. circulation) or move cdnify into
+    # an OPDS- or controller-focused class or file.
+    if not core_dir in sys.path:
```

Comment:
I'm not sure why these import problems from circ are only just popping up, but when referencing this file from circulation modules, `core` is not listed in the path. If there's a better way to do this, I'm happy to hear about it.